charts/bookkeeper/templates/config_map.yaml

```diff
@@ -0,0 +1,8 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ template "configmap.fullname" . }}
+data:
+  # Configuration values can be set as key-value properties
+  PRAVEGA_CLUSTER_NAME: pravega
```

Comment:
PRAVEGA_CLUSTER_NAME should not be hardcoded. It should be a value supplied by user in values.yaml

Reply:
done